distribution/docker/src/docker/Dockerfile

```diff
@@ -43,8 +43,13 @@ RUN chmod 0555 /bin/tini
 ################################################################################
 FROM ${base_image} AS builder
 
+<% if (docker_base == 'default' || docker_base == 'cloud') { %>
+# Install required packages to extract the Elasticsearch distribution
```

Comment:
We can move this comment outside this conditional so we don't have to duplicate it, yes?